apps/site/src/app/(main)/login/Login.tsx

```diff
@@ -16,6 +18,11 @@ async function Login({
 
 	return (
 		<div className="min-h-screen flex flex-col items-center justify-center">
+			<Image
+				src={clouds}
+				alt="Background clouds"
+				className="absolute top-0 w-[100vw] h-[200vh] z-[-1]"
```

Comment:
Issue: On anything lower than the lg, the image starts to squish, especially for mobile.

![image](https://github.com/user-attachments/assets/e10db8b2-8d3a-4f90-8e0b-28404b5f2c06)

Reply:
Oh yeah you can have the image cropped off at the sides on smaller screen widths.